test/Confluent.Kafka.UnitTests/Consumer.cs

```diff
@@ -0,0 +1,38 @@
+// Copyright 2016-2017 Confluent Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Refer to LICENSE for more information.
+
+using Xunit;
+using System;
+using System.Text;
+using System.Collections.Generic;
+using Confluent.Kafka.Serialization;
+
+
+namespace Confluent.Kafka.Tests
+{
+    public class ConsumerTests
+    {
+        [Fact]
+        public void Constuctor()
+        {
+            var config = new Dictionary<string, object>();
+            var e = Assert.Throws<ArgumentException>(() => { var c = new Consumer(config); });
+            Assert.True(e.Message.Contains("group.id"));
+            e = Assert.Throws<ArgumentException>(() => { var c = new Consumer<Null, string>(config, null, new StringDeserializer(Encoding.UTF8)); });
+            Assert.True(e.Message.Contains("group.id"));
```

Comment:
I didn't do any positive tests because 1. I didn't want to call into rd_kafka_new in the unit tests which don't require a broker to be running (though now I checked, I see that's fine even without specifying any config param other than group.id) and 2. every integration tests this case already. I want to leave that as is.

But I'll put in some comments.